lesson04/assignment/css/_hw4.scss

```diff
@@ -0,0 +1,117 @@
+h1 {
+	line-height: $h1-line-h;
+	 font-size: $h1-font-sz;
+	 color:  $h1-color;
+	 background-color: $h1-bcolor;
+	 text-align: $text-align;
+	 transform-origin: 50% 100%;
+	 text-shadow: 2px 2px 2px #333;
+	 opacity: 0;
+	 animation: fadeIn;
+	 animation-duration: 10s;
+	 animation-timing-function: ease-in-out;
+	 animation-fill-mode: forwards;
+	 animation-delay: .2s;
+ }
+ @keyframes fadeIn {
+	 from {
+		 opacity: 0;
+		 transform: scale(.5, .5, 1)
+	 }
+	 to {
+		 opacity: 1;
+		 transform: scale(1.5, 1.5, 2);
+	 }
+ }
+/*----------------- sliding picture --------------*/
+ img {
+	display: block;
+}
+
+.auto-slide {
+	/* outline: 1px solid red;*/
+	width: $img-w;
+	height: $img-h;
+	overflow: hidden;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.auto-slide__content {
+	list-style: none;
+	display: flex;
+	position: relative;
+	/* left: -1280px;  */
+	/*  transform: translateX(-640px); */
+	animation-name: slide;
+	animation-duration: 15s;
+	animation-iteration-count: infinite;
+	animation-timing-function: ease-in-out;
+	animation-direction: normal;
+}
+
+.auto-slide__content:hover {
+	animation-play-state: paused;
+}
+
+@keyframes slide {
+	10% { transform: translateX(0); }
+	15%,20% { transform: translateX(-640px); }
+	25%,30% { transform: translateX(-1280px); }
+	35%,40% { transform: translateX(-1920px); }
+	45%,50% { transform: translateX(-2560px); }
+	55%,60% { transform: translateX(-3200px); }
+	65%,70% { transform: translateX(-3840px); }
+	85%,90% { transform: translateX(-4580px); }
+	100% { transform: translateX(0); }
+}
+ /* ------  menu item: click menu to see price ----------   */
+  section {
+	 margin-top: 2%;
+	 font-size: $menu-font-sz;
+ }
+ .entrees {
+	float: left;
+	width: $menu-w;
+	height: $menu-h;
+	color: $menu-color;
+	background-color: $menu-bcolor;
+}
+ .entrees:hover {
+	 cursor: pointer;
+ }
+ .front{
+    @include menu-font;
+	transform: rotateY(0deg);
+	transition: transform 0.5s linear 0s;
+	backface-visibility: hidden;
+ }
+ .back{
+	@include menu-font;
+	transform: rotateY(180deg);
+	transition: transform 0.5s linear 0s;
+	backface-visibility: hidden;
+ }
+ .entrees:hover .front{ transform: rotateY(-180deg);}
+ .entrees:hover .back{ transform: rotateY(0deg);}
+ 
+ /* ---------  moving bubble from bottom to top ------------- */
+ .bubbles{
+	position: absolute;   bottom: 0;
```

Comment:
The bubbles are neat, but I'd be careful of the absolute positioning off of the main body that far (1000+px) as you get horizontal scrolling, perhaps moving the bubbles into their own container with position: relative and using left/right values.